l5kit/l5kit/data/map_api.py

```diff
@@ -176,6 +272,21 @@ def get_crosswalk_coords(self, element_id: str) -> dict:
 
         return {"xyz": xyz}
 
+    def is_traffic_light(self, element_id: str) -> bool:
+        """
+        Check if the element is a traffic light
```

Comment:
```suggestion
        Check if the given map element has traffic light field. If yes, the map element is controlled by a traffic light.
```

Reply:
no this tells you if the elements is a traffic light, there is no connection to a map element controlled by it